set.mm

```diff
@@ -369,13 +371,11 @@ useful for discussing objects (beyond simply true or false).  In particular,
     unambiguous 'klr 5';
   $)
 
-  $( Declare typographical constant symbols that are not directly used
-     in the formalism, but *are* symbols we find useful when
-     explaining the formalism. It is much easier to consistently use
-     a single typographical system when generating text. $)
+  $( Declare typographical constant symbols that are not directly used in the
+     formalism but are useful to explain it in comments. $)
 
-  $c & $.  $( Ampersand (read:  "and-also") $)
-  $c => $.  $( Double right arrow (read:  "proves") $)
+  $c & $.  $( Ampersand (read: "and"). $)
+  $c => $.  $( Double right arrow (read: "implies"). $)
```

Comment:
I'm not sure the change from "proves" to "implies" is valid, see i.e. https://us.metamath.org/mpeuni/dtrucor.html

Reply:
dtrucor illustrates the difference between the two kinds of implication at play:
* implication of formulas, ` ( ph -> ps ) `
* implication of theorems, ` |- ph => |- ps `.

If something is closer to "proves" or "it is provable that", it's rather the turnstile symbol itself ` |- `.

Reply:
Resolving this since there is another thread below on the same topic.